drivers/staging/vc04_services/bcm2835-audio/bcm2835.c

```diff
@@ -12,7 +12,7 @@
 #include "bcm2835.h"
 #include <soc/bcm2835/raspberrypi-firmware.h>
 
-static bool enable_hdmi, enable_hdmi0, enable_hdmi1;
+static bool enable_hdmi = true, enable_hdmi0, enable_hdmi1;
```

Comment:
This change isn't going to be accepted. Your overlay should be able to update the bootargs if necessary, but looking at the default DT files this appears to be set anyway.